src/mmw/apps/geoprocessing_api/validation.py

```diff
@@ -69,3 +71,104 @@ def validate_uuid(uuid):
         return False
 
     return uuid == str(u)
+
+
+def validate_gwlfe_prepare(data):
+    area_of_interest = data.get('area_of_interest')
+    wkaoi = data.get('wkaoi')
+    huc = data.get('huc')
+    layer_overrides = data.get('layer_overrides', {})
+
+    if not check_gwlfe_only_one([area_of_interest, wkaoi, huc]):
+        error = ('Invalid parameter: One and only one type of area object'
+                 ' (area_of_interest, wkaoi, or huc) is allowed')
+        raise ValidationError(error)
+
+    not_valid_layers = check_layer_overrides_keys(layer_overrides)
+
+    if not_valid_layers:
+        error = create_layer_overrides_keys_not_valid_msg(not_valid_layers)
+        raise ValidationError(error)
+
+    if ('__LAND__' in layer_overrides.keys() and
+            not check_land_layer_overrides(layer_overrides)):
+        error = create_layer_overrides_land_not_valid_msg(LAND_LAYER_OVERRIDES)
+        raise ValidationError(error)
+
+    if ('__STREAMS__' in layer_overrides.keys() is not None and
+            not check_streams_layer_overrides(layer_overrides)):
+        error = 'Invalid __STREAMS__ param: Must be `drb`, `nhdhr`, or `nhd`'
```

Comment:
Making this consistent with the other one:

```suggestion
        error = f'Invalid __STREAMS__ param: Must be one of {STREAM_LAYER_OVERRIDES}'
```